src/Test/Utilities/CommonTestBase.cs

```diff
@@ -100,7 +102,20 @@ static CommonTestBase()
             bool collectEmittedAssembly,
             bool verify);
 
-        private static readonly ImmutableArray<Emitter> s_emitters;
+        private static ImmutableArray<Emitter> s_emitters;
+
+        private static ImmutableArray<Emitter> Emitters
+        {
+            get
+            {
+                if (s_emitters.IsDefault)
+                {
+                    s_emitters = LoadEmitters();
```

Comment:
@pharring Yes i agree that would be correct code, I'm just unclear it's solving a problem.  Another way to say that is one year from now if a new developer asked us why that interlocked exists what would we say?  We were nervous this might break something so we did this instead?  If that's our reasoning under what circumstances can we ever remove that code?